charts/influxdb/Chart.yaml

```diff
@@ -1,23 +1,23 @@
 apiVersion: v1
 name: influxdb
-version: 4.8.3
+version: 4.8.4
 appVersion: 1.8.0
 description: Scalable datastore for metrics, events, and real-time analytics.
 keywords:
-- influxdb
-- database
-- timeseries
-- influxdata
+  - influxdb
+  - database
+  - timeseries
+  - influxdata
 home: https://www.influxdata.com/time-series-platform/influxdb/
 sources:
-- https://github.com/influxdata/influxdb
+  - https://github.com/influxdata/influxdb
 maintainers:
-- name: rawkode
-  email: rawkode@influxdata.com
-- name: gitirabassi
-  email: giacomo@influxdata.com
-- name: aisuko
-  email: urakiny@gmail.com
-- name: naseemkullah
-  email: naseem@transit.app
+  - name: rawkode
+    email: rawkode@influxdata.com
```

Comment:
I think you want to change this ?  And add @chobbs

Reply:
I will be adding to the charts today @timhallinflux